Frontend/ui-common/src/Components/ChooseVehicle/View.purs

```diff
@@ -0,0 +1,141 @@
+module Components.ChooseVehicle.View where
+
+import Components.ChooseVehicle.Controller (Action(..), Config)
+import Components.PrimaryButton as PrimaryButton
+import Data.Maybe (fromMaybe)
+import Data.String (Pattern(..), split, length)
+import Debug.Trace (spy)
+import Effect (Effect)
+import Engineering.Helpers.Commons (screenWidth, safeMarginBottom, os, flowRunner)
+import Font.Size as FontSize
+import Font.Style as FontStyle
+import Language.Types (STR(..))
+import Prelude (Unit,($),(<>),const,pure,unit)
+import PrestoDOM (Gravity(..), Length(..), Margin(..), Orientation(..), Padding(..), PrestoDOM, Visibility(..),  background, color, cornerRadius,  fontStyle,  gravity, height, imageView, letterSpacing, lineHeight, linearLayout, margin, onClick, orientation, padding, stroke, text, textSize, textView, visibility, weight, width, clickable, imageWithFallback , imageUrl)
+import PrestoDOM.Properties (cornerRadii)
+import PrestoDOM.Types.DomAttributes (Corners(..))
+import Screens.Types (Stage(..))
+import Styles.Colors as Color
+import Font.Size as FontSize
+import Font.Style as FontStyle
+import Common.Types.App
+
```

Comment:
@saireddy2610  Please revisit through the import statement, and remove unused

Reply:
kk